src/test/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/BankHolidaysClientTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.sendletter.services.date.holidays;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
```

Comment:
😱 . junit4 was disabled some time ago now. in order to run them we will need vintage dependency to be present. but best to find alternative to junit5 (register custom extension if needed)

Reply:
I we can achieve the same result with currently available jupiter annotations, we should do it.

Reply:
moved to junit5. the`@Rule` annotation could not be used